Palace/SignInLogic/TPPAccountSignInViewController.m

```diff
@@ -112,7 +112,7 @@ - (instancetype)init
 #endif
                         ];
 
-  self.title = NSLocalizedString(@"Sign In", nil);
+  self.title = NSLocalizedString(@"Sign in", nil).capitalizedString;
```

Comment:
Doesn't that mean that "i" will still be capitalised? I understand that it shouldn't be

Reply:
Nice catch, I misread the issue.